Flow_manager 23 test 070 failure

Hi,

The test test_070_install_flow in tests/test_e2e_23_flow_manager.py is failing:
```
self = <tests.test_e2e_23_flow_manager.TestE2EFlowManager instance at 0x7f73f96a4ab8>

    def test_070_install_flow(self):
        """
        Tests the performance and race condition with
        the creation of multiple flows using threading
        """
        import threading
        threads = list()
        for vlan_id in range(100, 200):
            t = threading.Thread(target=self.create_flow, args=(vlan_id,))
            threads.append(t)
        for thread in threads:
            thread.start()
        for thread in threads:
            thread.join()

        s1 = self.net.net.get('s1')
        flows_s1 = s1.dpctl('dump-flows')
>       assert len(flows_s1.split('\r\n ')) == 101
E       assert 2 == 101
E        +  where 2 = len([' cookie=0x0, duration=11.062s, table=0, n_packets=6, n_bytes=252, priority=1000,dl_vlan=3799,dl_type=0x88cc actions=...ation=1.069s, table=0, n_packets=0, n_bytes=0, priority=10,in_port="s1-eth1",dl_vlan=100 actions=output:"s1-eth2"\r\n'])
E        +    where [' cookie=0x0, duration=11.062s, table=0, n_packets=6, n_bytes=252, priority=1000,dl_vlan=3799,dl_type=0x88cc actions=...ation=1.069s, table=0, n_packets=0, n_bytes=0, priority=10,in_port="s1-eth1",dl_vlan=100 actions=output:"s1-eth2"\r\n'] = <built-in method split of str object at 0x55d7cf2b3f40>('\r\n ')
E        +      where <built-in method split of str object at 0x55d7cf2b3f40> = ' cookie=0x0, duration=11.062s, table=0, n_packets=6, n_bytes=252, priority=1000,dl_vlan=3799,dl_type=0x88cc actions=C...ration=1.069s, table=0, n_packets=0, n_bytes=0, priority=10,in_port="s1-eth1",dl_vlan=100 actions=output:"s1-eth2"\r\n'.split

tests/test_e2e_23_flow_manager.py:1085: AssertionError
```

This failing because after the flow creation it can take a couple of milliseconds to have the flows actually installed. So, we should insert a sleep before doing the dump-flows.

--- a/tests/test_e2e_23_flow_manager.py
+++ b/tests/test_e2e_23_flow_manager.py
@@ -1064,7 +1064,6 @@ class TestE2EFlowManager:
         data = response.json()
         assert 'FlowMod Messages Sent' in data['response']
 
-    @pytest.mark.xfail
     def test_070_install_flow(self):
         """
         Tests the performance and race condition with
@@ -1079,6 +1078,9 @@ class TestE2EFlowManager:
             thread.start()
         for thread in threads:
             thread.join()
+
+        # wait for the flow to be installed
+        time.sleep(10)
 
         s1 = self.net.net.get('s1')
         flows_s1 = s1.dpctl('dump-flows')

